tox.ini

```diff
@@ -0,0 +1,36 @@
+[tox]
+envlist = django{18,110,111}
```

Comment:
You probably want to explicitly mark this as py27-django so systems with default Py3 (like mine) won't use that instead.

Reply:
Yes - I will do so.